### PR TITLE
getting the hostname without a reverse DNS lookup

### PR DESCRIPTION
**Pull request by 2is10**
This fixes a `HostNotFoundException` I started encountering during local development today.

I checked and `hostname` seems to return the names we expect on production machines (e.g. `"shoebox-demand-1"`).

Please review @eishay or @andrewconner or @stkem.

### CODE REVIEW

**Comment by 2is10** [comment]
I got the idea from [Why does InetAddress.getLocalHost().getHostName() return a value different from bash “hostname”?](http://stackoverflow.com/questions/11143880/why-does-inetaddress-getlocalhost-gethostname-return-a-value-different-from) on StackOverflow.

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ case class ProdHealthCheckModule() extends HealthCheckModule {
 
   @Provides
   @AppScoped
-  def healthcheckHost(): HealthcheckHost = HealthcheckHost(InetAddress.getLocalHost.getHostName)
+  def healthcheckHost(): HealthcheckHost = HealthcheckHost("hostname".!!.trim)
```
Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala:20]: what's the `!!` syntax?

Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala:20]: Running something in the shell

Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala:20]: oh, fancy. forgot about this one

**Comment by stkem** [comment]
Not super familiar with the scala shell command tools, but from what I can tell this should be fine.

File: kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala (hunk 1 above)
Comment (2is10) [inline kifi-backend/modules/common/app/com/keepit/common/healthcheck/HealthCheckModule.scala:20]: Here’s the [scala.sys.process](http://www.scala-lang.org/files/archive/nightly/docs/library/#scala.sys.process.package) package documentation.

**Pull request merged**